data/remnant/remnant jobs.txt

```diff
@@ -334,7 +337,8 @@ mission "Remnant: Logistics Salvage Delivery"
 		dialog phrase "generic cargo on visit"
 	on complete
 		dialog `Two armed repair technicians inspects the <cargo> and authorize your payment of <payment>.`
-		payment 50000 25
+		payment
+		payment 10000
```

Comment:
For 50 cargo, this goes from 52500 to 25000.

Halving the average return for this one seems a bit of a big drop.

Reply:
Agreed. Maybe something like `payment 30000` instead at the end would make more sense?